airflow/dag_processing/manager.py

```diff
@@ -99,6 +102,13 @@ class DagFileStat:
 log = logging.getLogger("airflow.processor_manager")
 
 
+class DagFileInfo(NamedTuple):
+    """Information about a DAG file."""
+
+    path: str
```

Comment:
we should always try, somehow, whether through the variable name, or through docstring, to clarify whether this path is an absolute path, or a path relative to the bundle root.

Reply:
Left a comment, but there will be some further work in this area so I'll try and make sure it's more obvious then.